### PR TITLE
Retitle “Configuring kubelet Garbage Collection"

### LINKED ISSUE #25300: Retitle “Configuring kubelet Garbage Collection” and add incoming links
**This is a Bug Report**

**Problem:**
[Configuring kubelet Garbage Collection](https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/) is a page about performing a particular task, but it lives in the Concepts section. Tasks should be in the tasks section.

**Proposed Solution:**
- Change the title of the page https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/ to “Garbage collection for container images”
- Hyperlink from within https://kubernetes.io/docs/concepts/containers/ to the page https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/
(remember, this PR will just cover the English localization)

**Page to Update:**
https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/

**Additional Information:**
/kind cleanup
/language en
/sig node
/help

Relevant to issue #4510

It is OK to clean up the content of the page beyond what this issue describes.
- Check whether it makes sense to remove the [Deprecation](https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/#deprecation) section. This could be because we [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) or because the information is out of date.

### PR DESCRIPTION
**Pull request by Kartik494**
This PR fixes #25300

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/garbage-collection-container-images.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-title: Configuring kubelet Garbage Collection
+title: Garbage collection for container images
 content_type: concept
 weight: 70
 ---
 
 <!-- overview -->
 
-Garbage collection is a helpful function of kubelet that will clean up unused images and unused containers. Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
+Garbage collection is a helpful function of kubelet that will clean up unused [images](https://kubernetes.io/docs/concepts/containers/#container-images) and unused [containers](https://kubernetes.io/docs/concepts/containers/). Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
```
Comment (kbhawkey) [inline content/en/docs/concepts/cluster-administration/garbage-collection-container-images.md:10]: Hi @Kartik494 .
Do you want to link to the containers page from the GC page or the other way around?
Also, I suggest to write the like:
```
[containers](/docs/concepts/containers/)
```

If you change the name of the file (not sure if required), you will want to add a redirect for the original page?
See `website/static/_redirects`.

Comment (Kartik494) [reply content/en/docs/concepts/cluster-administration/garbage-collection-container-images.md:10]: Hi @kbhawkey i agreed i will be changing the title only and change the name of file to the original one  and i will be updated the link as you mentioned .
Thanks

**Comment by Kartik494** [comment]
Hi @kbhawkey  while i addressing the comments i observe that it took the domain name for github rather than k8s.io `(https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/cluster-administration/docs/concepts/containers/#container-images)` , so should i keep the link as  `(https://kubernetes.io/docs/concepts/containers/#container-images)`

File: content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-title: Configuring kubelet Garbage Collection
+title: Garbage collection for container images
 content_type: concept
 weight: 70
 ---
 
 <!-- overview -->
 
-Garbage collection is a helpful function of kubelet that will clean up unused images and unused containers. Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
+Garbage collection is a helpful function of kubelet that will clean up unused [images](docs/concepts/containers/#container-images) and unused [containers](docs/concepts/containers/). Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
```
Comment (kbhawkey) [inline content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md:10]: Hi @Kartik494 .
Would you add a forward slash before the link path:
`/docs/concepts/containers/`

Comment (Kartik494) [reply content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md]: Hi @kbhawkey i have addressed the comment as you suggested , but its again give the link address `https://github.com/kubernetes/website/blob/master/docs/concepts/containers/#container-images` not expected as k8s.io domain , could you please suggest!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25457--kubernetes-io-master-staging.netlify.app/docs/concepts/cluster-administration/kubelet-garbage-collection/

**Comment by kbhawkey** [comment]
Hi @Kartik494 . Looks great! ❇️ 
The links from the GC cluster-administration page to the container page seem fine.
@sftim, the issue suggested:

```
Hyperlink from within https://kubernetes.io/docs/concepts/containers/ to the page https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/
```
Instead, do you want to add a link **to** the GC page from this page:
`https://deploy-preview-25457--kubernetes-io-master-staging.netlify.app/docs/concepts/containers/#what-s-next`

**Pull request merged**